apps/web/app/(basenames)/api/name/[alreadyClaimedName]/route.ts

```diff
@@ -60,13 +65,19 @@ async function handler(req: NextApiRequest, res: NextApiResponse<ApiResponse>) {
       query: alreadyClaimedName,
       orComponentId: ownershipRegistryComponentId,
     });
-    res.status(200).json({ suggestion: JSON.parse(suggestion.response) as string[] });
-  } catch (e) {
-    if (e instanceof Error) {
-      logger.error('error generating suggestions', e);
-      res.status(500).json({ error: `failed to generate suggestions` });
+    return NextResponse.json<ApiResponse>({
+      suggestion: JSON.parse(suggestion.response) as string[],
+    });
+  } catch (error) {
+    if (error instanceof Error) {
+      logger.error('error generating suggestions', error);
     }
+    return NextResponse.json<ApiResponse>(
+      { error: `failed to generate suggestions` },
+      { status: 500 },
+    );
   }
 }
 
-export default withTimeout(handler);
+// TODO: add timeout
```

Comment:
Still necessary?

Reply:
nope, sorry. i had to amend the decorator, forgot to remove the comment